scene/__init__.py

```diff
@@ -69,6 +72,8 @@ def loop(self, lag_scalar, screen):
         # Start drawing this frame by painting the whole thing black
         screen.fill((0, 0, 0))
         current_scene.draw(screen)
+        screen_copy = screen.copy()
+        screen.blit(screen_copy, next(self.offset))
```

Comment:
Copying the screen buffer every frame is probably expensive.  Is there a way we can only do this when we're shaking the screen?

Reply:
Almost certainly. I'll try and lock this down.